File: app/src/main/java/com/dashingqi/master/MainActivity.java
package com.dashingqi.master;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.widget.TextViewCompat;
import androidx.databinding.DataBindingUtil;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;




public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        DataBindingUtil.setContentView(this, R.layout.activity_main);
        ButterKnife.bind(this);
//        mTextView.setText("hei,ha");
//        mTextView.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                RouteUtils.getInstance().jumpToActivity(MainActivity.this, "/test/test_activity");
//            }
//        });

    }
}

File: module-fragment/src/main/java/com/dashingqi/module/fragment/MainActivity.kt
package com.dashingqi.module.fragment

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private val fragments = ArrayList<Fragment>()

    private val mTitles = ArrayList<String>()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        for (index in 0 until 3) {
            fragments.add(TestFragment())
            mTitles.add("title$index")
            tabLayout.newTab().text = mTitles[index]
        }
        viewPager.adapter = TestFragmentAdapter(supportFragmentManager, fragments, mTitles)
        tabLayout.setupWithViewPager(viewPager)
    }
}
